build/building/env-vars.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Environment variables for Docker Build
+description: Use environment variables to configure builds
+keywords: docker build, buildx, buildkit, env, environment variables, config
```

Comment:
Should we add a redirect to this page as `/build/buildkit/color-output-controls/` is removed?

Reply:
yeah makes sense